.github/workflows/generate_api_docs.yaml

```diff
@@ -0,0 +1,39 @@
+name: "Generate openapi.json"
+
+env:
+  DEFAULT_PYTHON_VERSION: "3.10"
```

Comment:
Active support for 3.10 ended a year ago, any reason not to use the latest python version?